streams/src/test/java/org/apache/kafka/streams/tests/SmokeTestDriver.java

```diff
@@ -479,13 +483,23 @@ private static boolean verify(final PrintStream resultStream,
                 final Number expected = keyToExpectation.apply(key);
                 final Number actual = entry.getValue().getLast().value();
                 if (!expected.equals(actual)) {
-                    resultStream.printf("%s fail: key=%s actual=%s expected=%s%n\t inputEvents=%n%s%n\toutputEvents=%n%s%n",
-                                        topic,
-                                        key,
-                                        actual,
-                                        expected,
-                                        indent("\t\t", observedInputEvents.get(key)),
-                                        indent("\t\t", entry.getValue()));
+                    resultStream.printf("%s fail: key=%s actual=%s expected=%s%n", topic, key, actual, expected);
+
+                    if (printResults) {
+                        resultStream.printf("\t inputEvents=%n%s%n\t" +
+                                "echoEvents=%n%s%n\tmaxEvents=%n%s%n\tminEvents=%n%s%n\tdifEvents=%n%s%n\tcntEvents=%n%s%n\ttaggEvents=%n%s%n",
```

Comment:
Since some of the results like dif / tagg are composed of other results, just printing themselves are not sufficient for debugging, so I've decided to print all intermeidate results here too.